validation_utils.js

```diff
@@ -2,21 +2,33 @@
 "use strict";
 var chash = require('./chash.js');
 
+/**
+ * True if there is at least one field in obj that is not in arrFields.
+ */
 function hasFieldsExcept(obj, arrFields){
 	for (var field in obj)
 		if (arrFields.indexOf(field) === -1)
 			return true;
 	return false;
 }
 
+/**
+ * True if int is an integer as per ES6 Number.isInteger.
+ */
 function isInteger(int){
 	return (typeof int === 'number' && int.toString().indexOf('.') === -1 && !isNaN(int));
 }
 
+/**
+ * True if int is an integer strictly greater than zero.
+ */
 function isPositiveInteger(int){
 	return (typeof int === 'number' && int > 0 && int.toString().indexOf('.') === -1 && !isNaN(int));
```

Comment:
This should probably be:
```
return isInteger(int) && int > 0;
```

Reply:
i agree, will post a followup ticket later for this

Reply:
Cool - works for me